src/utils/helpers.ts

```diff
@@ -347,14 +348,20 @@ const helpers = {
 
     const outputs: Output[] = [];
     for (const output of data.outputs) {
-      const address = new Address(output.address);
-      const p2pkh = new P2PKH(address, { timelock: output.timelock || null });
-      const p2pkhScript = p2pkh.createScript()
-      const outputObj = new Output(
-        output.value,
-        p2pkhScript,
-        { tokenData: output.tokenData }
-      );
+      let outputObj;
+      if (output.data) {
```

Comment:
We should have at least the possibility to use multisig which we already support, but having support for custom output scripts would be nice as well.
Instead of having `output.type == 'data'`, why not `nft`, `p2pkh` `multisig` (or `p2sh`) and let data for custom scripts?

Reply:
Yeah, we support multisig in the protocol but adding support for it in the lib is more than just handling the output p2sh correctly. I think we should discuss this in another issue, we have a project for this in the future.